interface/reports/immunization_report.php

```diff
@@ -494,6 +517,33 @@ class='datepicker form-control'
             </div>
         <?php } ?>
     </form>
-
+    <script>
+        /*
+         * The better thing to do here would be to uncouple the query from this report
+         */
+        function exportData() {
+            let query = JSON.stringify("<?php echo $export_res; ?>");
+            let bindings = '<?php echo $export_bindings; ?>';
+            let csrf_token = '<?php echo CsrfUtils::collectCsrfToken(); ?>';
```

Comment:
for all 3 lines above, remove the surrounding quotes and pass them through js_escape() function
(and would then remove the serialize() call above on the $export_bindings function)